### PR TITLE
Home and about

### PR DESCRIPTION
**Pull request by Yilikal250**
i have implemented and created a web page that has a home page and a about page that is responsive and has the below features.

          2 pages:
   

- [ ]  The home page.
- [ ]     The about page.

 2 different screen sizes:

- [ ]     Mobile: up to 768px wide.
- [ ]     Desktop: 768px or wider.

Interactions

    Links
        

- [ ] The home page should have a link in the menu to the about page.
- [ ]         The logo in the header links to the home page.

    Mobile menu
       

- [ ]  When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
- [ ]         There are no guidelines for the mobile menu in the docs, but you should implement it so it is consistent with the design.
- [ ]         The mobile menu should have a close (X) button that closes the menu.

Dynamic page

    

- [ ] The section "Featured speakers" should be created dynamically in JavaScript.
- [ ]     You should use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+        <li> <a href="about.html">Program</a> </li>
+        <li> <a href="about.html">Join</a> </li>
+        <li>  <a href="about.html">Sponsor</a> </li>
+        <li>  <a href="about.html">News</a> </li>
+        <li id="bord"> <a href="about.html">CC Campain</a>  </li>
+        
+      </ul>
+     </div>
+     
+      <div><img class="logoimage" src="./image/logowithtxt.png" alt=""></div>
```
Comment (MahmoudBakr23) [inline index.html:53]: - [ ] Please make sure to align the logo image to the left side of the navbar to match the given design. 🙏

File: index.html [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+        </li>
+        <li><img class="img-logo" src="./image/naver.png" alt="" /></li>
+        <li><img class="img-logo" src="./image/airbnb.png" alt="" /></li>
+      </ul>
+    </div>
+
+    <div class="footer-contact">
+        <nav>
+        <p><a href="index.html"><img class="boot-img" src="./image/logowithtxt.png" alt=""></a></p>
+        <p id="place-ex">
```
Comment (MahmoudBakr23) [inline index.html:176]: - [ ] The same logo design issue here in the footer as well, please align it to the left side of the container. 🙏

File: README.md [hunk 3]
```diff
@@ -1 +1,57 @@
...
+> I created a website for a Good homes exihbition that has About and Home page with mobile verisons 
+    
+    A Good Homes exhibition home is a dwelling built for the purposes of the public exhibition and marketing of new dwellings,
+    whether or not it is intended to be sold as a private dwelling after its use for those purposes is completed, and includes 
+    any associated sales or home finance office or place used for displays.
+    
+    
+
+> this is a link to the loom video explaining about the project ( https://www.loom.com/share/d58a4f2d66974232bdbb0b765082b27e  )
+>  this is a link to the deployed page (  https://yilikal250.github.io/House-Expo/ )
```
Comment (MahmoudBakr23) [inline README.md:17]: - [ ] Please update your live demo link with the latest updates on your website. 👍

File: README.md [hunk 4]
```diff
@@ -1 +1,57 @@
...
+
+## Acknowledgments
+
+Microverse is the best place to learn !!
+
+
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.Original design idea by Cindy Shin in Behance.
```
Comment (MahmoudBakr23) [inline README.md:57]: - [ ] Please fix the MIT link for a completely professional README file.

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @Yilikal250 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Correct Linters
- Great use of JavaScript
- Well done following the given design so far
- Semantic HTML

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Kingstalux) [reply index.html]: resolved 👍

File: index.html (hunk 2 above)
Comment (Kingstalux) [reply index.html:177]: resolved 👍

File: README.md (hunk 3 above)
Comment (Kingstalux) [reply README.md]: resolved 👍

File: README.md (hunk 4 above)
Comment (Kingstalux) [reply README.md]: resolved 👍

**Review by Kingstalux — APPROVED** [review]
Hi @Yilikal250, 
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966

## Highlights ⚡ 
✅ Good UI design 👏 
✅ Great pull request description 👍 
✅ Good video presentation 🥂 

Great job implementing the requested changes 👍 Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :ship: 


Good luck! 🎉 

- [ ] (OPTIONAL) Kindly adjust your media queries so that from at least `1024px` the speaker section will have the desktop display. Right now your desktop display for the speaker's section starts at `1400px`. It would be great if you could fix this.
- Your design:
Annotation 2022-04-22 144952

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**